src/core/Cline.ts

```diff
@@ -94,6 +96,8 @@ export class Cline {
 		apiConfiguration: ApiConfiguration,
 		customInstructions?: string,
 		alwaysAllowReadOnly?: boolean,
+		alwaysAllowWrite?: boolean,
+		alwaysAllowExecute?: boolean,  // Add this line
```

Comment:
@hannesrudolph Your Cursor-IDE-added `// Add this line` is still appearing as a comment in this pull request.

The following suggestion — _should hopefully_ — remove the comment in this pull request.
```suggestion
		alwaysAllowExecute?: boolean,
```